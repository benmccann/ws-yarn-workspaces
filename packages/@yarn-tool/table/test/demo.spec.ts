import { createDependencyTable } from '..';

test(`createDependencyTable`, () =>
{

	let table = createDependencyTable();

	table.options.colAligns = ['left', 'center', 'center', 'center'];
	table.options.head = [
		'package name',
		'old version(s)',
		'',
		'new version(s)',
	];

	//expect(actual).toStrictEqual(expected);
	//expect(actual).toBeInstanceOf(Date);
	expect(table).toMatchSnapshot();

	expect(table.toString()).toMatchSnapshot();

});

