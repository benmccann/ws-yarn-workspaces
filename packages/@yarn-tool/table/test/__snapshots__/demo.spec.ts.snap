// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createDependencyTable 1`] = `Table []`;

exports[`createDependencyTable 2`] = `" package name  old version(s)    new version(s) "`;
